### PR TITLE
doc: cleanup HTTP Frontends documentation

### PR DESCRIPTION
**Pull request by falcon78921**
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Fixes: https://tracker.ceph.com/issues/37770
Added literal block to /etc/ceph/ceph.conf example.

Happy New Year! :fireworks: :)

Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

### CODE REVIEW

**Comment by falcon78921** [comment]
jenkins render docs

**Comment by tchaikov** [comment]
@falcon78921 thanks for the fixing! it'd be better if you could move the "Fixes: https://tracker.ceph.com/issues/37770" line right before your Signed-off-by line. so one who's reading your commit message could focus on the body of the commit message, and can find the "Fixes" line at the end of it.

and Happy New Year to you! =)

**Comment by falcon78921** [comment]
@tchaikov Fixed in 6f3100b. Thank you Kefu! :)

**Comment by tchaikov** [comment]
retest this please.

File: doc/radosgw/frontends.rst [hunk 1]
```diff
@@ -130,10 +130,13 @@ Options
...
 
-The following is an example of the ``/etc/ceph/ceph.conf`` file with some of these options set:
+The following is an example of the ``/etc/ceph/ceph.conf`` file with some of these options set: ::
  
  [client.rgw.gateway-node1]
- rgw frontends = civetweb request_timeout_ms=30000 error_log_file=/var/log/radosgw/civetweb.error.log access_log_file=/var/log/radosgw/civetweb.access.log
+ rgw_frontends = civetweb 
+ request_timeout_ms = 30000 
+ error_log_file = /var/log/radosgw/civetweb.error.log 
+ access_log_file = /var/log/radosgw/civetweb.access.log
```
Comment (cbodley) [inline doc/radosgw/frontends.rst]: sorry, i didn't catch this before it merged. all of these values need to be on the same line as part of the `rgw_frontends` value. this new config example won't produce the same result

Comment (tchaikov) [reply doc/radosgw/frontends.rst]: @cbodley sorry for this! will send a fix late tomorrow.

Comment (cbodley) [reply doc/radosgw/frontends.rst]: @tchaikov see https://github.com/ceph/ceph/pull/27054

Comment (falcon78921) [reply doc/radosgw/frontends.rst]: @tchaikov @cbodley My apologies for causing this error. If there's anything I can do fix, please let me know.

Comment (tchaikov) [reply doc/radosgw/frontends.rst]: @falcon78921 no worries. the upside is that @cbodley spotted this problem and we do have a fix =)

**Pull request merged**